connectors/src/connectors/google_drive/temporal/config.ts

```diff
@@ -1,3 +1,3 @@
 export const WORKFLOW_VERSION = 5;
-export const QUEUE_NAME = `google-queue-v${WORKFLOW_VERSION}`;
-export const GDRIVE_INCREMENTAL_SYNC_DEBOUNCE_SEC = 10;
+export const GDRIVE_FULL_SYNC_QUEUE_NAME = `google-queue-v${WORKFLOW_VERSION}`;
```

Comment:
Not renaming this queue because we have a workflow running on it for more than a week.
I have a task created to do that once the long full sync workflow is finished.
https://github.com/dust-tt/tasks/issues/929